### PR TITLE
Introduce the 2021 employee referral policy

### PR DESCRIPTION
**Pull request by adrianclay**

### CODE REVIEW

File: guides/hiring/referral_policy.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Employee Referral Policy
+
+The main purpose of this employee referral program policy is for our employees and all the
+People and Talent Team to use when they need detail and information about how Made Tech’s referral programs work.
+
+If you know someone who you think would be a good fit for a position at our company, feel free to refer them.
+If we end up hiring your referred candidate,
```
Comment (adrianclay) [inline guides/hiring/referral_policy.md:7]: This sentence appears to be missing an ending.

File: guides/hiring/referral_policy.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+This Employee Referral Program Policy applies to all our employees who refer candidate to our company.
+
+| Monetary Amount     | Salary Level (£) | Payment amt on start date | Payment amt after probation |
+| ------------------: | :--------------: | -------------------------:| --------------------------: |
+| £500                | 20,000 – 50,000  |                      £250 |                        £250 |
+| £1000               | 50,001 - 70,000  |                      £500 |                        £500 |
+| £1500               | 70,001 - 90,000  |                      £500 |                       £1000 |
+| £1750               | 90,001 - 125,000 |                      £750 |                       £1000 |
+
+__Charity__ – should you wish to not take us up on any of the exciting incentives then you can opt to make a charitable donation which we will make on your behalf to our associated charity for the given year of the campaign.
```
Comment (adrianclay) [inline guides/hiring/referral_policy.md:24]: It's unclear as to what charity this is.  Is this documented elsewhere, that could be linked to?

File: guides/hiring/referral_policy.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+> Recruiters and hiring managers for positions for which they’re hiring.
+
+## Who can be referred?
+
+Not all candidates can be referred. The only groups of people who cannot be referred include candidates:
+
+* That have applied in the past 6 months
+* Who were previously terminated from employment with us for any reason
+
+Thank you to all of our Made Tech employees for taking the time to consider their networks, social and professional as potential resources for referred candidates.
```
Comment (adrianclay) [inline guides/hiring/referral_policy.md:53]: This seems a continuation of the "Additional rules for rewards" section, so I suggest moving the sections "Who is eligible to take part?" and "Who can be referred?" before the "Additional rules" section.

File: guides/hiring/referral_policy.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+We may change our referral bonus program over time to add more interesting incentives.
+We also reserve the right to abolish certain rewards if they prove ineffective or inefficient.
+We’ll [communicate any change](https://resources.workable.com/employee-referral-bonus-program-announcement-sample-email) clearly and timely.
```
Comment (adrianclay) [inline guides/hiring/referral_policy.md:57]: I'm not sure this link is useful.

File: guides/hiring/referral_policy.md (hunk 1 above)
Comment (McLeanTech) [reply guides/hiring/referral_policy.md:7]: yes indeed it is missing some information. 

Should say: 'If we end up hiring your referred candidate that you may receive a referral payment. Use the policy guidelines below to determine your eligibility.'

File: guides/hiring/referral_policy.md (hunk 2 above)
Comment (McLeanTech) [reply guides/hiring/referral_policy.md:24]: Good point Adrian. We are yet to have a company associated charity. We have just hired a new People and Culture Manager who will be looking at this as part of their role. For now it perhaps makes sense for us to issue the payment to the individual and for them to make the donation to whomever they wish to. The benefit of having it in the policy like this is for tax implications so the charity can receive the most amount of money and company affiliated on the rationale its coming from Made Tech and ideally wants to be something that we as a company feel passionate about and can contribute to over the course of the year. I believe the P&C Manage starts next week 😁

File: guides/hiring/referral_policy.md (hunk 3 above)
Comment (McLeanTech) [reply guides/hiring/referral_policy.md:53]: agreed

File: guides/hiring/referral_policy.md (hunk 4 above)
Comment (McLeanTech) [reply guides/hiring/referral_policy.md:57]: definitely not helpful and not meant to be left in. We should remove and just say that we will communicate change in a clear and timely manner.

File: guides/hiring/referral_policy.md (hunk 2 above)
Comment (adrianclay) [reply guides/hiring/referral_policy.md:24]: 👍🏻 In that case, I'll create an issue as a reminder to come back to this at some point and leave the policy as it is for now. 🎉

Comment (adrianclay) [reply guides/hiring/referral_policy.md:25]: Documented in #437

File: guides/hiring/referral_policy.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+## Additional rules for rewards
+
+* Employees will receive awards within 30 days of each stage
+* We never say no to a receiving a profile of a referral and will ensure that someone from our
+TA team contacts that individual within 72hrs of receipt of the profile – you can refer as many time as you wish.
+Please be sure to evaluate our roles and requirements and make a judgement on suitability to ensure everyone’s time and expectations are well managed.
+* If the same candidate is referred by multiple employees, the first employee gets a reward, so act fast if you know someone that could be suited to our open positions.
+* There is a 12 month time limit on a referred candidate, meaning if you refer someone to us and they are not right for the current positions now, you could still receive a payment if they are hired in the next 12 months.
+You will however not receive a payment if you leave Made Tech.
+However the reward will align to the level set at the time of the introduction for our standard scheme (ie. Outside of promotional campaigns or revisions of this policy)
```
Comment (WillGibson) [inline guides/hiring/referral_policy.md:51]: My brain struggled to grok this. Does it mean...
```suggestion
* The amount of the reward will align to the level set at the time the introduction was made
```

Comment (McLeanTech) [reply guides/hiring/referral_policy.md:51]: Agree with the above from @WillGibson

**Comment by adrianclay** [comment]
For the PR approval police: <Approval from Micheal over Slack>

**Pull request merged**